packages/@yoda/network/test.js

```diff
@@ -0,0 +1,39 @@
+'use strict'
+
+var Network = require('index.js')
+var network = new Network()
+
+network.wifiClose().then((reply) => {
+  console.log(reply)
+}, (err) => {
+  console.log(err)
+})
+
+network.wifiOpen("ROKID.TC", "rokidguys").then((reply) => {
+  console.log(reply)
+}, (err) => {
+  console.log(err)
+})
+
+network.wifiStartScan().then((reply) => {
+  console.log(reply)
+}, (err) => {
+  console.log(err)
+})
+
+network.wifiStopScan().then((reply) => {
+  console.log(reply)
+}, (err) => {
+  console.log(err)
+})
+
+network.wifiScanList().then((reply) => {
+  console.log(reply)
+}, (err) => {
+  console.log(err)
+})
+
+setTimeout(() => {
+  console.log("finished")
+  process.exit(0)
+}, 20000)
```

Comment:
You need move the test to our `./test/@yoda/network` and use `tape` to assert, see https://github.com/yodaos-project/yodart/tree/master/test#yodaos-test-suite.

Reply:
OK